tools/packaging/kernel/build-kernel.sh

```diff
@@ -22,7 +22,6 @@ kernel_version=""
 # Flag know if need to download the kernel source
 download_kernel=false
 # The repository where kernel configuration lives
-readonly kernel_config_repo="github.com/${project_name}/kata-containers/tools/packaging"
 readonly patches_repo="github.com/${project_name}/kata-containers/tools/packaging"
```

Comment:
This one seems to also not be used in this script.

Would be worth doing a full check and remove all the unused vars at once.

Reply:
@fidencio changes applied :)